### ISSUE TITLE
GetSearch should not quote the requested query

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gyc2432968/python-twitter#150]
```
I found a bug in the GetSearch method. The 'q' parameter is quoted, which 
should not be done there. The parameters are quoted again later in the process.

E.g.:

GetSearch("#haiku") does not give the correct results. 

If you remove the quoting:

parameters['q'] = term

then it works fine for me.
```

Original issue reported on code.google.com by `huubhuub...@gmail.com` on 21 Jul 2010 at 2:19

**Comment by GoogleCodeExporter** [comment]
```
I've attached a patch.  extra_params will get urlencoded with urllib.urlencode 
in _EncodeParameters.  quote_plus on the term will cause "#haiku" to be 
incorrectly quoted as %2525haiku instead of just %25haiku.
```

Original comment by `blhob...@gmail.com` on 10 Sep 2010 at 3:31

Attachments:
- [remove_quote_plus.diff](https://storage.googleapis.com/google-code-attachments/python-twitter/issue-150/comment-1/remove_quote_plus.diff)

**Comment by GoogleCodeExporter** [comment]
```
fixed when revision 63651288b8 was committed
```

Original comment by `bear42` on 16 Oct 2010 at 7:43
- Changed state: **Fixed**
